### DISCUSSION TITLE
ScaleIntensityRanged - (a_min, a_max)

### DISCUSSION THREAD

**Discussion by HripsimeS** [Project-MONAI/tutorials#693] [Q&A] [answered]
Dear MONAI team,

I have a quick question about ScaleIntensityRanged. In your notebook you mentioned (a_min=-57, a_max=164) in train_transforms. 
https://github.com/Project-MONAI/tutorials/blob/master/3d_segmentation/spleen_segmentation_3d.ipynb

This choice was a random or you used especial method to find the optimal min/max intensity for the original image? Thanks in advance!

**Comment by Nic-Ma** [comment, marked as answer]
Hi @HripsimeS ,

`a_min=-57, a_max=164` is a typical value that only works for the spleen images.

Thanks.

**Reply by HripsimeS** [reply]
@Nic-Ma Hounsfield units help to calculate a correct intensity range, below is a description of calculation for each organ. This is a useful for other MONAI users if they want to understand how to chose a correct scale intensity range. 
https://radiopaedia.org/articles/windowing-ct

**Reply by wyli** [reply]
it's been answered already. https://github.com/Project-MONAI/MONAI/discussions/2583
